sw/device/silicon_creator/manuf/lib/BUILD

```diff
@@ -483,3 +483,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
+    name = "manuf_cp_test_lock_functest",
+    srcs = ["//sw/device/silicon_creator/rom/e2e:empty_test.c"],
+    cw310 = cw310_params(
+        bitstream = "//hw/bitstream:rom_otp_test_unlocked0",
```

Comment:
Use the OTP image I describe here: https://github.com/lowRISC/opentitan/pull/18471#discussion_r1191529819 instead as it will be completely empty, with ROM execution disabled. Just add it to #18471 and rebase this PR on top to use it too.

Reply:
Thanks, this with the bootstrapping removed fixed the errors I was having.